README.md

```diff
@@ -1,78 +1,83 @@
-- [Testing upgradeable applications](https://docs.zeppelinos.org/docs/testing.html)
-- [Extend provided EVM package code in your own contracts](https://github.com/zeppelinos/labs/tree/master/extensibility-study#extensibility-study) (experimental)
-- [Migrate your non-upgradeable legacy ERC20 token into an upgradeable version with an opt-in approach](https://docs.zeppelinos.org/docs/erc20_onboarding.html)
-- [Migrate your non-upgradeable legacy ERC20 token into an upgradeable version with a managed approach](https://github.com/zeppelinos/labs/tree/master/migrating_legacy_token_managed#migrating-legacy-non-upgradeable-token-to-upgradeability-with-managed-strategy) (experimental)
+And initialize the ZeppelinOS project:
 
+```sh
+zos init my-project
+```
 
-## Links
+Now it is possible to add contracts to the project with the `zos add` command,
+push these contracts to a blockchain network with `zos push`, use
+`zos create` to create instances for these contracts that later can be
+upgraded, and many more things.
 
-### Documentation
-- [ZeppelinOS](http://zeppelinos.org)
-- [Documentation site](https://docs.zeppelinos.org)
-- [ZeppelinOS Blog](https://blog.zeppelinos.org)
+Run `zos --help` for more details about thes and all the other functions of
+ZeppelinOS.
 
-### Code
-- [ZeppelinOS CLI (`zos`)](https://github.com/zeppelinos/zos/tree/master/packages/cli#readme)
-- [ZeppelinOS library (`zos-lib`)](https://github.com/zeppelinos/zos/tree/master/packages/lib#readme)
+The
+[ZeppelinOS documentation](https://docs.zeppelinos.org/)
+explains how to build a project using our platform, how to upgrade contracts,
+how to share packages for other projects to reuse, how to vouch for the quality
+of a package, how to use the JavaScript libraries to operate the project, and
+it explains details of the platform and some advanced topics.
 
 ## Security
 
-If you find a security issue, please contact us at security@zeppelinos.org. We give rewards for reported issues, according to impact and severity.
+If you find a security issue, please contact us at security@zeppelinos.org. We
+give rewards for reported issues, according to impact and severity.
+
+## Maintainers
 
-## Development setup
+* [@facuspagnuolo](https://github.com/facuspagnuolo/)
+* [@spalladino](https://github.com/spalladino)
 
-This monorepo is managed via [lerna](https://lernajs.io/). To get everything up and running, install lerna 3.x (either globally or locally from the root of the repository), and run `lerna bootstrap`. This will compile all packages, install all dependencies, and create symlinks for local dependencies (for instance, `packages/cli/node_modules/zos-lib` will be symlinked to `packages/lib`).
+## Contribute
 
-Note that, since `cli` and `lib` packages need to be precompiled, for any change in `lib` to impact `cli` you'll need to run a new `lerna bootstrap`.
+To contribute, join our
+[community channel on Telegram](https://t.me/zeppelinos) where you can talk to
```

Comment:
please validate with @martriay to make sure we are not missing any other channel